### ISSUE TITLE
doc: BrandBook: Visual Identity hurts my eyes

### ISSUE THREAD

**Issue by kaervin** [golang/go#25142]
Gone are the muted natural colors reminiscent of plan9,
only to be replaced by a flashy, saturated Visual Identity, trying to appeal to the hip and trendy crowd (or the new corporate world? I don't really understand.).

How could one even think this would fit the programming language go?
Isn't gos true Identity composed of those very simple, very muted colors? That sandy playground? That pale blue?
What would represent go better then those?

I hope that at least the color-choice of the documentation and the playground won't suffer for it in the end;
Because seriously, no joke: 
the new Visual Identity hurts my eyes.

**Comment by ianlancetaylor** [comment]
CC @spf13

**Comment by as** [comment]
This design is rather disappointing but there are a few observations I'll throw in here that will hopefully add more useful information than the unimportant sentiments of the _choir that no longer needs to be preached to_.

My largest hurdle in convincing people that Go is good isn't getting them to look past the classic look and feel of the brand--it's the reputation of the language being seen by other developers as "yet another Google product". When I ask them what that means, the answer conveys the belief of something fad-like that will lose steam and wither away when the next product/framework arrives (I don't follow Google products, or their reputations, so I don't know if this is true--but I believe this is a faithful approximation of the corpus of answers I've received from around 5-10 colleagues in the last few months).

People are actually interested in the history of Go and the origins of the language, the Plan 9 and Inferno lore associated with Go is part of that.  I don't know if it will make it more difficult for people to look past the brand and see why Go is unique in the first place, but the guide seems a bit coercive with what can and can't be done with "Go" when disseminating information about it in general.

**Comment by pciet** [comment]
The description looks like arbitrary “I don’t like change” to me.

> the new Visual Identity hurts my eyes.

Can you provide specific examples? Do you see utility problems?

On the slides the jumps between black and white seems harsh on my screen (eyes get used to black then the sun appears), but maybe that’s ok on a projector. I like the look overall.

**Comment by kaervin** [comment]
>The description looks like arbitrary “I don’t like change” to me.

My personal preference is for things not to change if there is no need to.
But in this case I don't even see one single reason for this change, and I think the change is completely for the worse.

Why do I think it's for the worse? Maybe I'll try to explain it more clearly:
- the colors are too saturated and garish (for my taste).
Rob wrote quite some time ago why he chose the colors he did for plan9, and I fully agree with his sentiment. (I would post a link, but I just can't find it. It's probably hidden somewhere on cat-v)
Paler, more natural colors are more relaxing to look at. And this is where it isn't just about my taste anymore.
The longer you stare at your screen, the more important this factor becomes.
And I personally do spend quite a while fixating my eyes on the docs or the playground.
So there is a _real reason_ for things to stay as they are that isn't just about personal preference.

- If it's about Identity, then this change doesn't represent the Identity of the go programming language at all!
I do like the logo (as long as the gopher stays). But in my mind, the 'Visual Identity' of go is undeniably what it currently is (or was?).
Dumping everything in a paint-bucket and telling the community "this is how things are now" just seems wrong to me.
Gos 'visual Identity' was already unmistakable and had a million times more charm than the one dictated by the Brand Book.
It represented the programming language and its roots perfectly. It remained simple. And isn't this pretty much the mission statement of this whole project?

For me it seems like someone at google thought Gos web-presentation didn't look "professional and stylish" enough (unlike every other web-anything it was simple and had charm), and so he decided that Go should be painted in Fuchsia, which is apparently the favorite color of whoever is in charge at google for this kind of stuff.

I am genuinely sad to see the 'old image' go. Especially when this change is just stupid and nobody from the community was asked about what they think.

**Comment by spf13** [comment]
Thank you @kaervin for your feedback. It is encouraging to see the passion you have for Go and we are grateful for you taking the time to share this with us.

I'd like to address your concerns the best I can, while recognizing that with matters like design, there is no way to appease everyone.  What we hear most clearly is a love of the gopher.  **The gopher has always been a beloved mascot for the brand and will continue to be.**

A big part of my role as the product manager for the Go language is doing research. I've conducted many surveys and many dozens of user feedback sessions. Some of the results of this research I've published on the [Go blog](https://blog.golang.org/survey2017-results).

As part of this research, I did user testing around our former brand and found that people consistently liked the gopher and how it brought a whimsical playfulness to the project, but beyond this there was not a lot of consistency in what people liked.

Using this feedback and working with Renee and many of the other Go team members along with a branding agency for many months we arrived at the brand launched this week. While it may seem like a drastic change to some people, most of what we did was take existing work and bring it together in a single canonical document.

Here are a few examples:
* The Colors we choose were inspired by the colors Renee had used for her Gophers over the years. 
* The Logo is a new design but is based on prior work that Renee had provided to the project 
![image](https://user-images.githubusercontent.com/173412/39436012-d80dfeca-4c6a-11e8-9f08-ed911b333661.png)
* The Values were taken from a variety of posts and presentations that Rob, Russ and others had authored. 
* The Mission & Vision were refinements to statements that have been made by the Go leadership for many years. 

Through this process we did continual user testing and made refinements along the way.

I understand and appreciate the difficulties that come from drastic changes, especially when you feel that the changes are deeper than just the look and feel. I hope this helps you understand why we made the changes we made and I also hope that you will continue to use Go with the passion you've brought here.
